src/consolidations/main.eas

```diff
@@ -213,7 +215,13 @@ read_requests:
   push MAX_PER_BLOCK    ;; [count, head_idx, tail_idx]
 
 begin_loop:
-  push0                 ;; [i, count, head_idx, tail_idx]
+  ;; Store request type prefix into the output buffer.
+  push1 REQUEST_TYPE    ;; [type, count, head_idx, tail_idx]
+  push 0                ;; [offset, type, count, head_idx, tail_idx]
+  mstore8               ;; [count, head_idx, tail_idx]
+
+  ;; Push initial loop index.
+  push 0                ;; [i, count, head_idx, tail_idx]
```

Comment:
```suggestion
  push0                ;; [i, count, head_idx, tail_idx]
```
don't forget to use our favorite op code :)

Reply:
But it will be used by default!

Reply:
ah that's right 👍